docs/source/graphs.md

```diff
@@ -1,117 +1,107 @@
 ---
 title: 'Working with non-federated graphs'
 sidebar_title: 'Non-federated graphs'
-description: 'How to use Rover to work with non-federated graphs'
+description: '(Graphs represented by a monolithic GraphQL server)'
 ---
 
-Rover comes equipped with many tools to help you manage your non-federated
-graph.
+> This article applies only to non-federated graphs. When working with a federated graph, see [Working with federated graphs](./subgraphs).
 
-> Everything on this page applies only to non-federated graphs. For a guide on
-> how to work with a federated graph, check out [Working with Subgraphs](./subgraphs).
+## Fetching a schema
+
+### Fetching from Apollo Studio
 
-## Introspecting Schemas
+> This requires first [authenticating Rover with Apollo Studio](./configuring/#authenticating-with-apollo-studio).
 
-Sometimes, you don't have a `.graphql` SDL file readily available. This is
-especially true with graphs built using certain libraries which don't use SDL
-to define schemas at all like `graphql-kotlin`.
+You can use Rover to fetch the current schema of any Studio graph and variant it has access to.
 
-For cases like this, or to download a schema from a publicly available endpoint,
-try using Rover's introspection command:
+Run the `graph fetch` command, like so:
 
 ```bash
-rover graph introspect http://my-public-api.com
+rover graph fetch my-graph@my-variant
 ```
 
-> TODO: headers API
+The argument `my-graph@my-variant` in the example above specifies the ID of the Studio graph you're pushing to, along with which [variant](https://www.apollographql.com/docs/studio/org/graphs/#managing-variants) you're pushing to.
```

Comment:
The `pushing to` looks like it may be a copy/pasta overlook. Is that right?

Reply:
Yeah, given the amount of copypasta involved in creating these two articles, I may one day see whether I can merge them without creating confusion